mne/evoked.py

```diff
@@ -996,22 +996,33 @@ def grand_average(all_evoked, interpolate_bads='eeg'):
     return grand_average
 
 
-def merge_evoked(all_evoked):
-    """Merge/concat evoked data
+def combine_evoked(all_evoked, weights='nave'):
+    """Merge evoked data by weighted addition
 
     Data should have the same channels and the same time instants.
+    Subtraction can be performed by passing negative weights (e.g., [1, -1]).
 
     Parameters
     ----------
     all_evoked : list of Evoked
-        The evoked datasets
+        The evoked datasets.
+    weights : list of float | str
```

Comment:
I like passing ones better.  Seems more explicit.  It's not no weighting, it's equal weighting.